documentation/slate/source/includes/welcome.md

```diff
@@ -143,10 +143,11 @@ Want to know if your city is in Navitia? Know if a special contributor is used?
 Getting help
 ------------
 
-All available functions are documented in [integration part](#interface) . If you want to go further, there is an OpenAPI/swagger documentation > https://api.navitia.io/v1/schema
-
 >[Try openAPI / swagger](https://api.navitia.io/v1/schema)
```

Comment:
It would be nice to integrate it to a rendering tool.
But when I tried https://petstore3.swagger.io/ I bumped into 2 issues:

1. Don't know how to provide "deep_link"
2. This "invalid {...}" part at the bottom of the page :sob: